lib/plugins/config/admin.php

```diff
@@ -105,53 +107,68 @@ public function html()
             } elseif ($setting instanceof SettingFieldset) {
                 // config setting group
                 if ($in_fieldset) {
-                    echo '</table>';
                     echo '</div>';
-                    echo '</fieldset>';
+                    echo '</section>';
                 } else {
                     $in_fieldset = true;
                 }
                 if ($first_plugin_fieldset && $setting->getType() == 'plugin') {
-                    $this->printH1('plugin_settings', $this->getLang('_header_plugin'));
+                    $this->printHeadline(kHeadlineLevel, 'plugin_settings', $this->getLang('_header_plugin'));
                     $first_plugin_fieldset = false;
                 } elseif ($first_template_fieldset && $setting->getType() == 'template') {
-                    $this->printH1('template_settings', $this->getLang('_header_template'));
+                    $this->printHeadline(kHeadlineLevel, 'template_settings', $this->getLang('_header_template'));
                     $first_template_fieldset = false;
                 }
-                echo '<fieldset id="' . $setting->getKey() . '">';
-                echo '<legend>' . $setting->prompt($this) . '</legend>';
-                echo '<div class="table">';
-                echo '<table class="inline">';
+                echo '<section id="' . $setting->getKey() . '">';
+                echo '<h3>' . $setting->prompt($this) . '</h3>';
```

Comment:
TBH, I wonder if this is actually more semantic. (not that I care much either way)

Reply:
While `<div>` is a _generic_, block-level element without semantics, `<section>` is specifically for marking up, well, page- or article-_sections_. I would say that "section" is the best way to describe the blocks in this place. Of course, the same could be achieved by giving the div a `role` attribute, but there is no need to go this route, if there is a suitable HTML element readily available.